packages/shared/components/atoms/tiles/tileContents/TransactionActivityTileContent.svelte

```diff
@@ -0,0 +1,197 @@
+                classes="overflow-hidden overflow-ellipsis multiwrap-line2"
+            >
+                {localize(title)}
+            </Text>
+            <Text
+                fontWeight={FontWeight.semibold}
+                lineHeight="140"
+                color={isIncoming ? 'blue-700' : ''}
+                classes="whitespace-nowrap"
+            >
+                {amount}
+            </Text>
+        </div>
+        <div class="flex flex-row justify-between">
+            <Text fontWeight={FontWeight.medium} lineHeight="140" color="gray-600">
+                {localize(isIncoming ? 'general.fromAddress' : 'general.toAddress', {
+                    values: { account: subjectLocale },
+                })}
+            </Text>
+            <Text fontWeight={FontWeight.medium} lineHeight="140" color="gray-600" classes="whitespace-nowrap">
+                {fiatAmount}
+            </Text>
+        </div>
+    </div>
+</info-container>
+{#if isAsyncActivity}
+    <HR />
+    <async-activity-container class="flex w-full justify-between space-x-4">
+        <info-container class="flex flex-row justify-center items-center space-x-2">
+            {#if isUnclaimed || isTimelocked}
+                <TooltipIcon
+                    icon={isTimelocked ? IconEnum.Timelock : IconEnum.ExpirationTime}
+                    iconClasses="text-gray-600 dark:text-gray-200"
+                    title={localize(`general.${isUnclaimed ? 'expirationTime' : 'timelockDate'}`)}
+                    text={asyncStatusTooltipText}
+                    position={Position.Top}
+                />
+                <Text fontSize="13" color="gray-600" fontWeight={FontWeight.semibold}>{timeDiff}</Text>
+            {/if}
+        </info-container>
+        <claim-container class="flex flex-row justify-end w-1/2 space-x-2">
+            {#if isTimelocked}
+                <Pill backgroundColor="gray-200" darkBackgroundColor="gray-200">
+                    {localize('pills.locked')}
+                </Pill>
+            {:else if isIncomingActivityUnclaimed}
+                <Button
+                    onClick={handleRejectClick}
+                    disabled={data.isClaiming || data.isRejected}
+                    inlineStyle="min-width: 4rem;"
+                    size={ButtonSize.Small}
+                    outline
+                >
+                    {localize('actions.reject')}
+                </Button>
+                <Button
+                    onClick={handleClaimClick}
+                    disabled={data.isClaiming}
+                    isBusy={data.isClaiming}
+                    inlineStyle="min-width: 4rem;"
+                    size={ButtonSize.Small}
+                >
+                    {localize('actions.claim')}
+                </Button>
+            {:else}
+                <ActivityAsyncStatusPill asyncStatus={data.asyncStatus} />
+            {/if}
+        </claim-container>
+    </async-activity-container>
+{/if}
```

Comment:
This can also be extracted into its own component specifically for async activities